inst/include/Rcpp/api/meat/module/Module.h

```diff
@@ -23,12 +23,12 @@
 namespace Rcpp {
 
     inline List Module::classes_info(){
-	    int n = classes.size() ;
+	    size_t n = classes.size() ;
```

Comment:
I'm thinking more like `https://github.com/RcppCore/Rcpp/pull/556/files#diff-a1af5c14c54d6d6907a0ca6f99d61286R69` (although that's `int` to `R_xlen_t`, so I misspoke earlier)